qt/scientific_interfaces/Indirect/IndirectFitAnalysisTab.cpp

```diff
@@ -99,6 +99,9 @@ void IndirectFitAnalysisTab::connectDataPresenter() {
 void IndirectFitAnalysisTab::connectPlotPresenter() {
   connect(m_plotPresenter.get(), SIGNAL(fitSingleSpectrum(WorkspaceID, WorkspaceIndex)), this,
           SLOT(singleFit(WorkspaceID, WorkspaceIndex)));
+  connect(m_plotPresenter.get(), SIGNAL(startXChanged(double)), this, SLOT(startXChanged(double)));
+  connect(m_plotPresenter.get(), SIGNAL(endXChanged(double)), this, SLOT(endXChanged(double)));
```

Comment:
These seem to have the same calls for both the Signal and the Slot - could this get confusing?

Reply:
Thanks for renaming these